coaster-blas/Cargo.toml

```diff
@@ -15,20 +15,20 @@ keywords        = ["blas", "coaster", "computation", "hpc", "plugin"]
 license         = "MIT OR Apache-2.0"
 
 [dependencies]
-coaster = { version = "^0.1", default-features = false }
-lazy_static = "^1.0.0"
+coaster = { version = "0.1", default-features = false }
+lazy_static = "1"
 
 # native
-rblas = { git = "https://github.com/spearow/rust-blas", optional = true }
+rblas = { version = "0.0", optional = true }
```

Comment:
Shouldn't this be `rust-blas = { version = "0.1", optional = true }`? (even though it requires some renaming)

Reply:
Yep, you're right.